src/plugins/vis_type_timeline/server/series_functions/worldbank_indicators.js

```diff
@@ -83,9 +82,11 @@ export default new Datasource('worldbank_indicators', {
       return worldbank.timelineFn(wbArgs, tlConfig);
     });
 
-    return Bluebird.map(seriesLists, function (seriesList) {
-      return seriesList.list[0];
-    }).then(function (list) {
+    return Promise.all(
+      seriesLists.map(function (seriesList) {
+        return seriesList.list[0];
+      })
```

Comment:
nit
```suggestion
      seriesLists.map(({ list }) => list[0])
```

Reply:
Thanks for the nit. I've fixed this in local and push. Would like to commit yours but looks like it'll get flagged as ESLint issue :((